rio_tiler/utils.py

```diff
@@ -41,6 +41,12 @@ def aws_get_object(
     if not client:
         session = boto3_session()
         endpoint_url = os.environ.get("AWS_S3_ENDPOINT", None)
+        if endpoint_url is not None:
```

Comment:
we should add some comments at least in the code saying that `AWS_S3_ENDPOINT` and `AWS_HTTPS` are GDAL configuration options.

Reply:
Is it what you had in mind?

Reply:
yes